test/js-api/instanceTestFactory.js

```diff
@@ -360,6 +360,30 @@ const instanceTestFactory = [
     }
   ],
 
+  [
+    "export i64-returning function",
+    function() {
+      const builder = new WasmModuleBuilder();
+
+      builder
+        .addFunction("fn", kSig_l_v)
+        .addBody([
+            kExprI64Const, 0x66,
+            kExprReturn,
+        ])
+        .exportFunc();
+
+      const buffer = builder.toBuffer();
+
+      return {
+        buffer,
+        args: [],
+        exports: {},
+        verify: instance => assert_equals(instance.exports.fn(), -26n)
```

Comment:
Shouldn't the result be 102n?

Reply:
We call [`signed_64()`](https://webassembly.github.io/spec/core/exec/numerics.html#aux-signed), so the high bit is a sign bit.